### PR TITLE
fix: Update PHP for all and nodejs for the builders.

### PR DESCRIPTION
**Pull request by cafuego**
So that we satisfy minimum requirements for the common design.

Refs: CD-437

### CODE REVIEW

File: php/builder8/Dockerfile.tmpl [hunk 1]
```diff
@@ -28,9 +28,9 @@ LABEL org.label-schema.schema-version="1.0" \
       info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json mcrypt memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sodium sysvmsg sysvsem sysvshm simplexml sockets xml xmlreader xmlwriter xsl zip zlib" \
       info.humanitarianresponse.php.sapi="cli" \
       info.humanitarianresponse.composer="2.1.5" \
-      info.humanitarianresponse.node="12.18" \
-      info.humanitarianresponse.npm="12.18" \
-      info.humanitarianresponse.yarn="1.7.0" \
+      info.humanitarianresponse.node="16.19.0" \
+      info.humanitarianresponse.npm="8.19.3" \
+      info.humanitarianresponse.yarn="1.22.19" \
```
Comment (rupl) [inline php/builder8/Dockerfile.tmpl:33]: purely curious: you had some console output in the ticket that said npm 9.x but these images have 8.x (which I did mention in the ticket). Is that on purpose?

Either way I'm happy to bump them all the way up to whatever is latest, or stick with 8.x for the time being. We do tend to sit on node/npm versions for a while, so if I had to pick one, I'd go with npm 9.

File: build.sh [hunk 2]
```diff
@@ -140,7 +140,7 @@ pushd php/php-k8s-v81-NR && \
 
 # Build the php 8.1 builder image.
 pushd php/builder81 && \
-  make VERSION=${VERSION81} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=16-alpine build && \
+  make VERSION=${VERSION81} EXTRAVERSION=${EXTRAVERSION} UPSTREAM=16.19.0-alpine build && \
```
Comment (teodorescuserban) [inline build.sh:143]: is this the latest alpine (3.17) and the older builder just needs 3.16 or is it a typo?

**Review by teodorescuserban — DISMISSED** [review]

File: php/builder8/Dockerfile.tmpl (hunk 1 above)
Comment (cafuego) [reply php/builder8/Dockerfile.tmpl:33]: I must have just pasted wrong :-)

File: build.sh (hunk 2 above)
Comment (cafuego) [reply build.sh:143]: This is the latest 3.17, which only has PHP8.1. The PHP8.0 needs the older Alpine 3.16.

File: php/builder8/Dockerfile.tmpl [hunk 3]
```diff
@@ -28,9 +28,9 @@ LABEL org.label-schema.schema-version="1.0" \
       info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json mcrypt memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sodium sysvmsg sysvsem sysvshm simplexml sockets xml xmlreader xmlwriter xsl zip zlib" \
       info.humanitarianresponse.php.sapi="cli" \
       info.humanitarianresponse.composer="2.1.5" \
-      info.humanitarianresponse.node="12.18" \
-      info.humanitarianresponse.npm="12.18" \
-      info.humanitarianresponse.yarn="1.7.0" \
+      info.humanitarianresponse.node="16.19.0" \
+      info.humanitarianresponse.npm="8.19.3" \
```
Comment (cafuego) [inline php/builder8/Dockerfile.tmpl]:
```suggestion
      info.humanitarianresponse.npm="9.3.0" \
```

File: php/builder81/Dockerfile.tmpl [hunk 4]
```diff
@@ -28,9 +28,9 @@ LABEL org.label-schema.schema-version="1.0" \
       info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sodium sysvmsg sysvsem sysvshm simplexml sockets xml xmlreader xmlwriter xsl zip zlib" \
       info.humanitarianresponse.php.sapi="cli" \
       info.humanitarianresponse.composer="2.1.5" \
-      info.humanitarianresponse.node="12.18" \
-      info.humanitarianresponse.npm="12.18" \
-      info.humanitarianresponse.yarn="1.7.0" \
+      info.humanitarianresponse.node="16.19.0" \
+      info.humanitarianresponse.npm="8.19.3" \
```
Comment (cafuego) [inline php/builder81/Dockerfile.tmpl]:
```suggestion
      info.humanitarianresponse.npm="9.3.0" \
```

File: php/builder8/Dockerfile.tmpl (hunk 1 above)
Comment (rupl) [reply php/builder8/Dockerfile.tmpl:33]: Ok 👍  I went with 8.19.3

**Pull request merged**